cmake/RunLint.cmake

```diff
@@ -1,18 +1,27 @@
-get_filename_component(LINT_DIR ${LINT_DIR} ABSOLUTE)
-get_filename_component(LINT_PREFIX ${LINT_DIR} PATH)
+get_filename_component(LINT_DIR "${LINT_DIR}" ABSOLUTE)
+get_filename_component(LINT_PREFIX "${LINT_DIR}" PATH)
```

Comment:
@jszakmeister Thanks for explaining. It's more predictable and less surprising, then, which is desirable.